src/recon_buildblock/distributable.cxx

```diff
@@ -423,7 +423,7 @@ void distributable_computation(
 #ifdef STIR_OPENMP
 #pragma omp single
     {
-      std::cerr << "Starting loop with " << omp_get_num_threads() << " threads\n"; 
+      info("Starting loop with " + std::to_string(omp_get_num_threads()) + " threads");
```

Comment:
sorry. will fail as `to_string` is C++-11 onwards, and we currently still support older C++.
Please use `boost::format` as in other places